newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Personally, I prefer internal consistency over strictly following PEP8 - in that sense PEP8 is more of a guideline. Hence, If we're leaving the `import logging...` pattern it might be good to do this once and for all in a single pull request rather than have it be part of other work and creating an intermittent inconsistent state which leaves new contributers to wonder which convention to follow.

In other words; I have a slight personal preference for staying with the initial convention of initializing loggin on the top, but am willing to let this go. In that case I would really appreciate if you could do a search & replace for all such cases in all files so at least we're left with internal consistency.

PEP8 should already be handled as part of Landscape, which also tests for stuff like complexity, so nothing needs to be done there.